src/libs/OptionsListUtils.js

```diff
@@ -291,6 +292,7 @@ function createOption(logins, personalDetails, report, reportActions = {}, {
         result.isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
         result.shouldShowSubscript = result.isPolicyExpenseChat && !report.isOwnPolicyExpenseChat && !result.isArchivedRoom;
         result.brickRoadIndicator = getBrickRoadIndicatorStatusForReport(report, reportActions);
+        result.pendingAction = report.pendingFields ? report.pendingFields.addWorkspaceRoom : null;
```

Comment:
😕 what is `addWorkspaceRoom` doing here? Isn't this a generic component?

Reply:
> I added a pendingAction key to the options for use with the OfflineWithFeedback component. For now I set the pendingAction only for adding the workspace room, but it can be updated to be a combination of more fields in the future.

That's from my description of the PR. Yes createOption is called several times by getOptions, which is used by getSearchOptions, getNewChatOptions, and getMemberInviteOptions. What's the harm in setting a pendingAction?

Reply:
My confusion comes from: we only need to show RBR in the LHN, no? Wouldn't this make it show in all those other places too?

Reply:
We need to show the RBR / 50% opacity in the LHN and the search. It won't apply for a new chat, and I don't think it will apply for the member invite options, but I haven't confirmed that. Thanks for raising this point, because I should look into that. I will test / explain whether it applies for the member invite or not.

Reply:
> createOption is called several times by getOptions, which is used by getSearchOptions, getNewChatOptions, and getMemberInviteOptions.

The `pendingAction` should be set for rooms when calling getSearchOptions.

getNewChatOptions only returns reports with a single participant. It skips options without a login.https://github.com/Expensify/App/blob/2df16aa4ece79a39f0e7ac63c518a3172a361219/src/libs/OptionsListUtils.js#L546-L549

The login is only set when creating the option if the report doesn't have multiple participants.
https://github.com/Expensify/App/blob/2df16aa4ece79a39f0e7ac63c518a3172a361219/src/libs/OptionsListUtils.js#L364-L368

hasMultipleParticipants is always true for chatRooms.
https://github.com/Expensify/App/blob/2df16aa4ece79a39f0e7ac63c518a3172a361219/src/libs/OptionsListUtils.js#L317

For getMemberInviteOptions we do not pass reports to getOptions so we won't access any rooms.
https://github.com/Expensify/App/blob/2df16aa4ece79a39f0e7ac63c518a3172a361219/src/libs/OptionsListUtils.js#L742-L756

So in summary we will only have a pendingAction for rooms in the LHN and search, as intended.